src/objectTranslations/Account-de.objectTranslation

```diff
@@ -1,13 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-    <fields>
-        <help>Der Batch, in dem dieser Account erstellt wurde</help>
-        <label><!-- Batch --></label>
+    <fieldSets>
+        <label><!-- BDE Entry FS --></label>
```

Comment:
@force2b QQ, Michael, when the translations are entered in like this with <!-- Text -->, how does it show up in the actual UI? Are these labels user-facing? Will the users see the labels in English like this in the product?

Reply:
Good question. This is actually how Salesforce's defaults all untranslated items in metadata. So, in this case the fieldset for "BDE Entry FS" shows as untranslated (i.e., English) in the UI.